grace/io/image_dataset.py

```diff
@@ -86,6 +123,41 @@ def __getitem__(self, idx: int) -> tuple[torch.Tensor, dict]:
         return image, target
 
 
+def relabel_unknown_labels(G: nx.Graph):
```

Comment:
This is perfectly ok, but I just wanted to highlight that there is a networkx method to do this maybe more efficiently? https://networkx.org/documentation/stable/reference/generated/networkx.relabel.relabel_nodes.html

Reply:
I've looked at this function, but the mapping allows you to use new names for the node_idx, not for node_attributes. Here, what I need to do is to iterate over all nodes & edges to relabel their `GraphAttrs.GROUND_TRUTH` to `Annotation.TRUE_NEGATIVE`. Is there a handy function to do this simpler?

Reply:
I see, i think then your approach is the best one :)